### ISSUE TITLE
The name of the game can not be changed.

### ISSUE THREAD

**Issue by Shahin3d** [TheGameCreators/GameGuruRepo#232]
The name of the game can not be changed.
Upon completion of the construction of the final game (save standalone) can not change the name of the game.
And Game Guru name can be seen in Windows.
Is there a way to change the name of Game Guru and have my own game name?

![the name of the game can not be changed](https://user-images.githubusercontent.com/36647058/40069268-0c50a216-5880-11e8-9d3e-8a4a5dfa0589.jpg)

**Comment by Shahin3d** [comment]
Please fix this item sooner in the new update
My game is over and I want to have my own game name.

**Comment by Teabone3** [comment]
Not a bug but certainly a feature I think a lot of us would love to have. There are ways to change the process name I believe.

This method I'm assuming is only local and doesn't change the process name if you are distributing:
https://www.techwalla.com/articles/how-to-change-a-process-name-on-windows-task-manager

The old me would create include a .BAT launcher file. Another way is to HEX edit the EXE file.
https://www.youtube.com/watch?v=nHpEcOqYOzQ

**Comment by Shahin3d** [comment]
See for example: Unity Build setting

![unity build setting](https://user-images.githubusercontent.com/36647058/40573818-aaa06500-60dc-11e8-932f-d3f1f82e625b.png)

**Comment by Shahin3d** [comment]
It is considered a risk to my game
Bad people may abuse this issue
If they become aware --- that was made with GG !!! ???
They may sabotage or manipulate ...

**Comment by Shahin3d** [comment]
But this is a difficult task and does not answer forever
Have a permanent and substantial work to do forever
For example:
When Save standalone
Instead of the name Game Guru
Place the selected stage name.

**Comment by Shahin3d** [comment]
Has anyone checked this out?

**Comment by Shahin3d** [comment]
I made this change
Of course, with the software: HxD and ResourceHacker
I changed the name, description, and icon of the game.
I have no problem.(Issues232)
I did it just for my game.
I created a game and I want to sell it.
I wanted to have my group's name and logo.
I thank GG.
Because they allowed it.

-------------------------------------------------- --------
Of course, the GG team can add as a new feature:
A new setting for EXE output
Such as the Unity engine or other engines ...

![untitled](https://user-images.githubusercontent.com/36647058/41486013-06e1f876-70f8-11e8-8139-fd4d2821bdf3.png)
